exchanges/binance/binance_test.go

```diff
@@ -2490,12 +2491,42 @@ func TestSetExchangeOrderExecutionLimits(t *testing.T) {
 }
 
 func TestWsOrderExecutionReport(t *testing.T) {
-	t.Parallel()
+	// cannot run in parallel due to inspecting the DataHandler result
+	// t.Parallel()
 	payload := []byte(`{"stream":"jTfvpakT2yT0hVIo5gYWVihZhdM2PrBgJUZ5PyfZ4EVpCkx4Uoxk5timcrQc","data":{"e":"executionReport","E":1616627567900,"s":"BTCUSDT","c":"c4wyKsIhoAaittTYlIVLqk","S":"BUY","o":"LIMIT","f":"GTC","q":"0.00028400","p":"52789.10000000","P":"0.00000000","F":"0.00000000","g":-1,"C":"","x":"NEW","X":"NEW","r":"NONE","i":5340845958,"l":"0.00000000","z":"0.00000000","L":"0.00000000","n":"0","N":null,"T":1616627567900,"t":-1,"I":11388173160,"w":true,"m":false,"M":false,"O":1616627567900,"Z":"0.00000000","Y":"0.00000000","Q":"0.00000000"}}`)
+	expRes := order.Detail{
+		Price:           52789.1,
+		Amount:          0.00028400,
+		Exchange:        "Binance",
+		ID:              "5340845958",
+		ClientID:        "c4wyKsIhoAaittTYlIVLqk",
+		Side:            order.Buy,
+		Type:            order.Limit,
+		Status:          order.New,
+		AssetType:       asset.Spot,
+		Pair:            currency.NewPair(currency.BTC, currency.USDT),
+		RemainingAmount: 0.000284,
+	}
+	// empty the channel. otherwise mock_test will fail
+	for len(b.Websocket.DataHandler) > 0 {
+		<-b.Websocket.DataHandler
+	}
+
 	err := b.wsHandleData(payload)
 	if err != nil {
 		t.Fatal(err)
 	}
+	res := <-b.Websocket.DataHandler
+	switch r := res.(type) {
+	case *order.Detail:
+		// ignore the date
+		r.Date = time.Time{}
+		if diff := cmp.Diff(expRes, *r); diff != "" {
```

Comment:
I would suggest using `if !reflect.DeepEqual(expRes, *r)` here . For what is being compared between the response and the expected response, it achieves the same purpose without a new single use package import

Reply:
fine with me, it will not highlight as nicely what is different, but I understand that this might not be worth adding another dependency unless other tests will use it as well.